### PR TITLE
add missing system library required for azure ml python packages

### PR DESCRIPTION
**Pull request by YunchuWang**
Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

Adding missing linux system library liblttng-ust0 required by azure ml python packages for native support. Library install size is less than 800kb and has been asked in multiple incidents, stackoverflow and forums.
[[BUG] AzureML on Azure Functions Consumption Plan Linux · Issue #1124 · Azure/azure-functions-python-worker · GitHub](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-functions-python-worker%2Fissues%2F1124&data=05%7C01%7Cprnikabakshi%40microsoft.com%7C96a8a4c8d2d443583aec08dad1372a02%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638052332117249766%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=icyNvG%2Bc7qCf7mJmPpqDsqitqoHDPpyfzxe1QoI8R0Y%3D&reserved=0)

-[Incompatibility between linux python azure function and azure ML · Issue #1007 · Azure/azure-functions-python-worker · GitHub](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-functions-python-worker%2Fissues%2F1007&data=05%7C01%7Cprnikabakshi%40microsoft.com%7C96a8a4c8d2d443583aec08dad1372a02%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638052332117249766%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2Fi3Yt2nV%2BpCubaQha3vY9M3Dnz3UxGVtbFeWFVccUI%3D&reserved=0)

-[python - Azure ML error: NotImplementedError: Linux distribution debian 11. does not have automatic support - Stack Overflow](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstackoverflow.com%2Fquestions%2F74181670%2Fazure-ml-error-notimplementederror-linux-distribution-debian-11-does-not-have&data=05%7C01%7Cprnikabakshi%40microsoft.com%7C96a8a4c8d2d443583aec08dad1372a02%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638052332117249766%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=R2jaMtPFxGfcqHhq07c3%2FxMyNnJQ4ASyOQoC5pPeszQ%3D&reserved=0)
### PR information

- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Comment by YunchuWang** [comment]
Checked how much size increase this library install adds, taking an example (py310-slim image):
dev branch py310 slim total image size 1.5GB
image
layer containing missing library size 334mb
image

pr branch py310 slim total image size 1.5GB
image
layer containing missing library size 336mb
image

total size increase of this library install is less than 2mb.

File: host/4/bullseye/amd64/python/python310/python310-composite.template [hunk 1]
```diff
@@ -54,7 +54,9 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # mysql dependencies
-    apt-get install -y default-libmysqlclient-dev
+    apt-get install -y default-libmysqlclient-dev && \
+    #  Azure ML dependencies
+    apt-get install -y liblttng-ust0
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python310/python310-composite.template:59]: You need to include the && \ to continue the run statement on the next line. I believe. I will check out the builds

File: host/4/bullseye/amd64/python/python310/python310.Dockerfile [hunk 2]
```diff
@@ -89,7 +89,9 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # mysql dependencies
-    apt-get install -y default-libmysqlclient-dev
+    apt-get install -y default-libmysqlclient-dev && \
+    #  Azure ML dependencies
+    apt-get install -y liblttng-ust0
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python310/python310.Dockerfile:94]: This one is fine without the continuation chars && \ as it is the final command

File: host/4/bullseye/amd64/python/python310/python310-slim.Dockerfile [hunk 3]
```diff
@@ -84,7 +84,9 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157
-    apt-get install -y libc-dev
+    apt-get install -y libc-dev && \
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python310/python310-slim.Dockerfile:87]: This is the correct format.

File: host/4/bullseye/amd64/python/python37/python37-composite.template [hunk 4]
```diff
@@ -47,9 +47,12 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # mysql dependencies
-    apt-get install -y default-libmysqlclient-dev
+    apt-get install -y default-libmysqlclient-dev && \
+    #  Azure ML dependencies
+    apt-get install -y liblttng-ust0
     # Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157
-    #apt-get install -y libc-dev
+    #apt-get install -y libc-dev && \
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python37/python37-composite.template:54]: remove continuation

File: host/4/bullseye/amd64/python/python37/python37-composite.template [hunk 5]
```diff
@@ -47,9 +47,12 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # mysql dependencies
-    apt-get install -y default-libmysqlclient-dev
+    apt-get install -y default-libmysqlclient-dev && \
+    #  Azure ML dependencies
+    apt-get install -y liblttng-ust0
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python37/python37-composite.template:52]: Add continuation

**Review by CooperLink — COMMENTED** [review]
Please update the syntax for valid builds. Comments are all about && \

File: host/4/bullseye/amd64/python/python310/python310-composite.template (hunk 1 above)
Comment (YunchuWang) [reply host/4/bullseye/amd64/python/python310/python310-composite.template:59]: image
last lines are commented out

**Review by CooperLink — APPROVED** [review]

**Pull request merged**